src/components/AccountsPage/AccountRowItem/index.js

```diff
@@ -104,14 +104,25 @@ const mapDispatchToProps = {
   openModal,
 }
 
+const expandedStates: { [key: string]: boolean } = {}
+
 class AccountRowItem extends PureComponent<Props, State> {
-  state = {
-    expanded: false,
+  constructor(props) {
+    super(props)
+    const { account, parentAccount } = this.props
+    const accountId = parentAccount ? parentAccount.id : account.id
```

Comment:
why is there a need for this, aren't expand state only used for the parent case anyway, so basically just account.id case is needed?

Reply:
uhmm I'll check, you are probably right, I assumed without a state change it wouldn't trigger a render @gre